### PR TITLE
EXPANDR-1610 Qualys integration - Add "details" arg to qualys-host-list command

### PR DESCRIPTION
**Pull request by capanw**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue
https://jira-hq.paloaltonetworks.local/browse/EXPANDR-1610

## Description
A few sentences describing the overall goals of the pull request's commits.
Add details arg to _qualys-host-list_ command in _Qualys integration_ to pull owner related details on a host that is scanned.

## Screenshots
image

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/qualys/ReleaseNotes/1_2_3.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Qualys v2
+- Command ***qualys-host-list***: Added optional arguments ***details***
```
Comment (johnnywilkes) [inline Packs/qualys/ReleaseNotes/1_2_3.md:4]: you need to update this to follow one of the prefixes: https://xsoar.pan.dev/docs/documentation/release-notes#general

File: Packs/qualys/Integrations/Qualysv2/Qualysv2.yml [hunk 2]
```diff
@@ -546,6 +546,17 @@ script:
...
     - description: Specify a positive numeric value to limit the amount of results in the requested list
       name: limit
+      auto: PREDEFINED
+    - auto: PREDEFINED
+      defaultValue: Basic
+      description: '(Optional) Show the requested amount of host information for each host. A valid value is: Basic, Basic/AGs, All, All/AGs, or None.'
+      name: details
+      predefined:
+      - Basic
+      - Basic/AGs
```
Comment (johnnywilkes) [inline Packs/qualys/Integrations/Qualysv2/Qualysv2.yml:556]: Can you specify somewhere (maybe in description) that AG = Assets Groups, please

**Review by johnnywilkes — COMMENTED** [review]
some comments

File: Packs/qualys/ReleaseNotes/1_2_3.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+#### Integrations
+
+#### Qualys v2
+Added a details arg to qualys-host-list command.
```
Comment (daryakoval) [inline Packs/qualys/ReleaseNotes/1_2_3.md:4]:
```suggestion
Added *details* arg to the ***qualys-host-list*** command.
```

File: Packs/qualys/Integrations/Qualysv2/Qualysv2.yml (hunk 2 above)
Comment (capanw) [reply Packs/qualys/Integrations/Qualysv2/Qualysv2.yml:556]: Updated

**Comment by daryakoval** [comment]
Hi, please update me on when I can merge it.

**Comment by capanw** [comment]
Hello @daryakoval, I am done with the changes. You can go ahead and please merge it. Thank you.

**Review by daryakoval — APPROVED** [review]

**Pull request merged**